index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Challenge - Ian Grantham</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- overall grid -->
+    <section class="grid">
+
```

Comment:
Ill try clone your work later to see it visually because from seeing your code. This looks really really good well structured and organised I cant wait to see it on live